Packs/JOB-GetAtRiskIncidents/Scripts/AtRiskIncidentsNotification/AtRiskIncidentsNotification.py

```diff
@@ -0,0 +1,201 @@
+import datetime
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+context = demisto.context()
+found_incidents = context["foundIncidents"]
+# incidentbucket={"Red":[],"Orange":[],"Purple":[],"Maroon":[],"Olive":[],"Blue":[]}
+incidentbucket = {"Red": [], "Orange": [], "Purple": [], "Blue": []}
+# print(found_incidents)
+# print(type(found_incidents))
+
+if isinstance(found_incidents, list):
+    for inc in found_incidents:
+
+        temp_dict = {}
+        temp_dict[inc["id"]] = "https://sandma.demisto.live/#/Custom/caseinfoid/" + inc["id"]
+
+        containsladuedate = inc['CustomFields']['containsla']['dueDate']
+        # print(containsladuedate)
+        if containsladuedate == "0001-01-01T00:00:00Z":
+            containsladuedate = "2021-00-01T01:03:27.456384423Z"
+            containsladuedate = containsladuedate.split(".")[0]
+        else:
+            containsladuedate = containsladuedate.split(".")[0]
+
+        resolutionsladuedate = inc['CustomFields']['resolutionsla']['dueDate']
+        if resolutionsladuedate == "0001-01-01T00:00:00Z":
+            resolutionsladuedate = "2021-09-01T01:03:27.456384423Z"
+            resolutionsladuedate = resolutionsladuedate.split(".")[0]
+        else:
+            resolutionsladuedate = resolutionsladuedate.split(".")[0]
+
+        containsladuedatedt = datetime.datetime.strptime(containsladuedate, "%Y-%m-%dT%H:%M:%S")
+        resolutionsladuedatedt = datetime.datetime.strptime(resolutionsladuedate, "%Y-%m-%dT%H:%M:%S")
+
+        timenow = datetime.datetime.utcnow()
+
+        containbreach = -1
+        resolutionbreach = -1
+        if not timenow > containsladuedatedt:
+            containbreach = round((containsladuedatedt - timenow).seconds / 60)
+            print(containbreach)
```

Comment:
it' s preferable not to use print in the code (this place and other places).
one more thing: (not an obligation just looks better) could you please remove the comments from the code?